byu-calendar-row/byu-calendar-row.scss

```diff
@@ -152,17 +153,20 @@ $ticket-region-width: 250px;
 	}
 
 	#month-name {
-		padding: 10px 0px;
```

Comment:
I think originally this was because of a weird bug with firefox handling paddings in a weird way. But if you change the padding a little bit, and remove the height specification, then it seems to work just fine. I'll include a patch after I've reviewed everything here.